userapps/devshell/devshell.h

```diff
@@ -88,7 +94,7 @@ struct linux_dirent64 {
 static inline int
 getdents64(unsigned fd, struct linux_dirent64 *dirp, unsigned count)
 {
-   return sysenter_call3(220, fd, dirp, count);
+   return sysenter_call3(SYS_getdents64, fd, dirp, count);
```

Comment:
Same comment as for the other one. I'm OK to using the macro, if you add an ASSERT checking that on i386, SYS_getdents64 is actually 220.

Reply:
fixed!